meinberlin/apps/budgeting/assets/BudgetingProposalList.jsx

```diff
@@ -56,6 +53,9 @@ export const BudgetingProposalList = (props) => {
                     onPaginate={newUrl => onPaginate(newUrl)}
                   />}
               </>
+              )
+            : (
+              <div>{django.gettext('Nothing to show')}</div>
```

Comment:
@khamui the snapshots always show this version, never the empty list version. I have tried a number of ways to change that but i'm still stumped, maybe we can have a look together?